docs/install/installEverest.md

```diff
@@ -144,4 +138,25 @@ To install and provision Percona Everest:
     ./everestctl install operators --backup.enable=false --everest.endpoint=http://127.0.0.1:8080 --operator.mongodb=true --operator.postgresql=true --operator.xtradb-cluster=true --monitoring.enable=true --monitoring.type=pmm --monitoring.new-instance-name=my-pmm --monitoring.pmm.endpoint=http://127.0.0.1 --monitoring.pmm.username=admin --monitoring.pmm.password=admin  --skip-wizard
     ```
 
+    ### Limitations
+        
+    * If the Everest CLI fails to install the operators, do the following:
+        
+        * Remove the [EKS](../quickstart-guide/eks.md#remove-the-eks-cluster) or [GKE](../quickstart-guide/gke.md#remove-the-gke-cluster) cluster.
```

Comment:
Imo it would be best to guide to the everest uninstallation instead of deleting EKS cluster because it will take one hour

Reply:
updated